### DISCUSSION TITLE
How to use different table for diff domain query?

### DISCUSSION THREAD

**Discussion by EricKong1985** [zilliztech/GPTCache#519] [General]
I get a document from offical, 
https://gptcache.readthedocs.io/en/latest/bootcamp/langchain/baby_agi.html
Could you help to guide? how to different table for diff domain query?
Since I diff domain by I have just one server!

onnx = Onnx()
cache_base = CacheBase('sqlite')
vector_base = VectorBase('milvus', host='127.0.0.1', port='19530', dimension=onnx.dimension)
data_manager = get_data_manager(cache_base, vector_base)
cache.init(
    pre_embedding_func=get_content_func,
    embedding_func=onnx.to_embeddings,
    data_manager=data_manager,
    similarity_evaluation=SearchDistanceEvaluation(),
    )
cache.set_openai_key()

**Comment by SimFG** [comment]
like
```
cache_base = CacheBase('sqlite', table_name="cache1")
vector_base = VectorBase('milvus', host='127.0.0.1', port='19530', dimension=onnx.dimension, collection_name="cache1")
```
